yt_dlp/extractor/fujitv.py

```diff
@@ -19,7 +19,8 @@ class FujiTVFODPlus7IE(InfoExtractor):
         'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
         'info_dict': {
             'id': '5d40810075',
-            'title': '5d40810075',
+            'title': 'ちびまる子ちゃん　#1317 『おっちゃんのまほうカード』の巻／『まるちゃん おばけ屋敷にいく』の巻',
+            'description': '【原作35周年！あなたの好きな“神回”さくらももこ原作まつり】\n明日から夏休み。まる子は、一学期の荷物をいっきにひきずりながら歩いていた。下校途中の木の下で、おじさんが不思議なカードを売っているのを見かけたまる子とお姉ちゃんは、どうしても欲しくなり…／お母さんにデパートでやっているおばけ屋敷に連れていってとせが むまる子。あっさり却下されるが、お父さんが実は自分も行きたかったと連れて行ってくれることに。いよいよおばけ屋敷に入ったまる子と父ヒロシ。次々あらわれるおばけに二人の運命は…？',
```

Comment:
md5 this
`'description': 'md5:<md5 hash>'

Reply:
Also change the title to remove the series name (and fill in the series field)